cmd/ctr/run_unix.go

```diff
@@ -121,6 +122,9 @@ func newTask(ctx gocontext.Context, client *containerd.Client, container contain
 			io = containerd.StdioTerminal
 		}
 		if nullIO {
+			if tty {
+				return nil, errors.New("tty and null-io are exclusive")
```

Comment:
I would actually just say that `"tty and null-io cannot be used together"`  saying they "are exclusive" is a little vague.